ntc_templates/templates/arista_eos_show_ip_mroute_vrf_all_detail.textfsm

```diff
@@ -0,0 +1,42 @@
+Value Filldown VRF (\S+)
+Value Filldown MULTICAST_GROUP (\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})
+Value Required RPF_SOURCE (\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})
+Value RP (\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})
+Value UPTIME (\d+d\d+h|\d+:\d+:\d+)
+Value FLAGS (\S+)
+Value INCOMING_INTERFACE (\S+)
+Value UPSTREAM_JOINED_STATE (\S+)
+Value UPSTREAM_RPT_JOINED_STATE (\S+)
+Value List OUTGOING_INTERFACE_LIST (\S+)
+
+Start
+  ^VRF:${VRF}
+  ^PIM
+  ^RPF
+  ^\s+M\s-
+  ^Flags
+  ^\s{4}[RWIHZANTVF]\s-
+  ^(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}) -> Continue.Record
+  ^${MULTICAST_GROUP}
+  ^\s{2}(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}) -> Continue.Record
+  ^\s{2}${RPF_SOURCE},\s${UPTIME},\sflags:\s${FLAGS}
+  ^\s{2}${RPF_SOURCE},\s${UPTIME},\sRP\s${RP},\sflags:\s${FLAGS}
+  ^\s{4}Incoming\sinterface:\s${INCOMING_INTERFACE}
+  ^\s{4}Outgoing\sinterface\slist:
+  ^\s{6}${OUTGOING_INTERFACE_LIST}$$
+  ^\s{4}Interfaces\snot\sin\sOIL
+  ^\s{6}\S+
+  ^\s{4}\S\s-> Continue
```

Comment:
I'm going through this more closely and will provide updated regexes (most likely in a PR to your fork).

There are several regexes that are loose if it wasn't for the curly brace quantifiers.
And there's a syntax error, but that doesn't seem to cause the template to fail since it is mistaken as a pattern.

Reply:
@jmcgill298 and I used those loose regexes as an escape to the capture of the `OUTGOING_INTERFACE_LIST` because there was no pattern to match on in those lines to break out of that section without capturing the lines in the State summarization macros section below it. The line that matches that continue statement would be re-evaluated below. I will say, however that I put in 27 and 28 after the fact, and I did mess up the order of it there. 29 should actually be sitting above 27, directly below `^\s{6}${OUTGOING_INTERFACE_LIST}$$`. I will correct that and resubmit.

Reply:
The loose `^\s{6}\S+` and changes to its whitespace were the crux of issues (that I then caused).

I've managed to remove all the curly brace quantifiers and keep the template parsing. :sweat_smile: